### PR TITLE
add firefox profile settings to prevent saving dialog (for https://github.com/DevExpress/testcafe/issues/4869)

### PR DESCRIPTION
**Pull request by AlexKamaev**
we need to check the assumption that the `Download` dialog in FF affects the further tests execution

I use https://www.npmjs.com/package/firefox-profile for creating FF profile.
FF profile is passed to browserstack throught the `capabilities['firefox_profile']` option.
The value of the option is zip-archive with the profile file converted to binary.

The `firefox-profile` module can create a zip-archive from JS-object, then read it. After that we can pass the binary to capabilities.

However, this module has some predefined capabilities https://github.com/saadtazi/firefox-profile-js/blob/8c1b45fc4004a790ab28c86ba4d6536b76258f07/lib/firefox_profile.js#L20)

It's not clear why these settings are default, and it's not clear do we need them.
The other moment, should we use OUR settings defined in TestCafe.

At this moment we think about releasing the alpha version with only the `browser.helperApps.neverAsk.saveToDisk` setting

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -168,6 +179,37 @@ export default {
...
+
+        profile.setPreference('browser.helperApps.neverAsk.saveToDisk', getMimeTypes());
+        profile.updatePreferences();
+
+        capabilities['firefox_profile'] = await this._encodeFirefoxProfile(profile);
+    },
+
+    async _encodeFirefoxProfile (profile) {
+        return new Promise((resolve, reject) => {
+            profile.encoded(function (err, encodedProfile) {
```
Comment (miherlosev) [inline src/index.js]: I guess we can use the `util.promisify` here.
```js
async _encodeFirefoxProfile (profile) {
   const promisifiedProfileEncoded = util.promisify(profile.encoded);
  
   return promisifiedProfileEncoded(); 
}
```

Or combine call as following:
```js
capabilities['firefox_profile'] = await util.promisify(profile.encoded)();
```

File: src/index.js [hunk 2]
```diff
@@ -197,8 +239,11 @@ export default {
         if (!capabilities.name)
             capabilities.name = `TestCafe test run ${id}`;
 
-        if (browserName.indexOf('chrome') !== -1 && process.env['BROWSERSTACK_CHROME_ARGS'] && process.env['BROWSERSTACK_CHROME_ARGS'].length > 0)
-            capabilities.chromeOptions = { args: [process.env['BROWSERSTACK_CHROME_ARGS']] };
+        if (browserName.indexOf('chrome') !== -1)
```
Comment (miherlosev) [inline src/index.js]: `if(browserName.includes('chrome')`)

File: src/index.js [hunk 3]
```diff
@@ -197,8 +239,11 @@ export default {
...
+            this._prepareChromeCapabilities(capabilities);
+
+        if (browserName.indexOf('firefox') !== -1)
```
Comment (miherlosev) [inline src/index.js]: `if (browserName.includes('firefox'))`

**Comment by AlexKamaev** [comment]
@AndreyBelym has already approved this PR

**Pull request merged**